workers/data_refinery_workers/downloaders/management/commands/queue_test_task.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
+from data_refinery_models.models import (
+    SurveyJob,
+    Batch,
+    BatchStatuses,
+    DownloaderJob
+)
+from data_refinery_workers.downloaders.array_express \
+    import download_array_express
+
+
+# Import and set logger
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+# Just a temporary way to queue a celery task
+# without running the surveyor.
```

Comment:
I went ahead and made that change. But thanks for linking that! I had read it at some point in the past but I guess I misremembered seeing it in pep8 so when I went looking for it I couldn't find it.